monitoredresource/monitored_resources_test.go

```diff
@@ -101,11 +101,3 @@ func TestAWSEC2InstanceMonitoredResources(t *testing.T) {
 		t.Errorf("AWSEC2InstanceMonitoredResource Failed: %v", autoDetected)
 	}
 }
-
-func TestNullMonitoredResources(t *testing.T) {
```

Comment:
This was failing on Travis because (I guess) the Travis worker is an EC2 instance.

Reply:
It's actually a GCE instance :) Tests failed because GCE resource is detected correctly, which is a good indication.